daffodil-core/src/main/scala/org/apache/daffodil/grammar/primitives/PrimitivesZoned.scala

```diff
@@ -137,8 +119,8 @@ abstract class ConvertZonedNumberPrim[S](e: ElementBase)
         case TextNumberRounding.Pattern => (MaybeDouble.Nope, Nope)
       }
 
-    val nff = new NumberFormatFactoryStatic[S](
-      e.termRuntimeData, h,
+    val ev = new TextNumberFormatEv(
```

Comment:
This change set is going to fix numerous outstanding cleanup tickets. 

The thought that crosses my mind is whether you want to follow this change set up with a separate commit that just does file-reorganization of the resulting code. I am often tempted to do that but then don't because it makes diffs for review problematic, but if you do it as two separate PRs, one of which is only the file reorg, that's less an issue.

Reply:
Yeah, i'll undo the renames so that the diffs make sense, and we can do that in a later commit. There are probably a bunch more than I didn't notice that need to be renamed.